content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-jenkinsfile.md

```diff
@@ -239,6 +245,12 @@ The account `project-admin` needs to be created in advance since it is the revie
    input(id: 'release-image-with-tag', message: 'release image with tag?', submitter: 'project-admin,project-admin1')
    ···
    ```
+   
+   {{< notice note >}}
+   
+   In KubeSphere v3.1, only pipeline creator, account with the role of admin in the project, or the account you specify will have the authority to continue or terminate the pipeline.
```

Comment:
There're two cases here:

* First, anyone who can run this pipeline, then he can approve or reject it if there are no specific approvers
* Second, same as you described above

Reply:
Thanks you the comment and I made some changes.